### PR TITLE
Added validation for high and low value fields for add item page

### PR DESCRIPTION
**Pull request by Manan-dev**
# Description

-> Added validation for high and low value fields for add item page.
-> Added dollar sign adornment

## Relevant issue(s)

#117 
## Type of change

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] I have assigned reviewers to this PR

### CODE REVIEW

**Review by AlanKha — DISMISSED** [review]

**Comment by AlanKha** [comment]
Looks good to me

File: src/views/addItemView/index.tsx [hunk 1]
```diff
@@ -1,15 +1,46 @@
 'use client';
-import { Box, Button, Grid, TextField, Typography } from '@mui/material';
+import TopBar from '@/components/top-bar';
```
Comment (alay10) [inline src/views/addItemView/index.tsx:2]: Not used

File: src/views/addItemView/index.tsx [hunk 2]
```diff
@@ -98,41 +129,68 @@ export default function AddItemView() {
...
-                label="Low Value"
-                variant="outlined"
-                fullWidth
-                margin="normal"
-                value={lowValue}
-                onChange={(e) =>
-                  setLowValue(Number(e.target.value) < 0 ? '0' : e.target.value)
-                }
-              />
+              <FormControl fullWidth sx={{ m: 1 }}>
```
Comment (alay10) [inline src/views/addItemView/index.tsx:160]: Unless you added this for a reason, the margin should probably stay 0

File: src/views/addItemView/index.tsx [hunk 3]
```diff
@@ -98,41 +129,68 @@ export default function AddItemView() {
...
-                fullWidth
-                margin="normal"
-                value={highValue}
-                onChange={(e) =>
-                  setHighValue(
-                    Number(e.target.value) < 0 ? '0' : e.target.value
-                  )
-                }
-              />
+              <FormControl fullWidth sx={{ m: 1 }}>
```
Comment (alay10) [inline src/views/addItemView/index.tsx:132]: Same on this line

**Review by alay10 — CHANGES_REQUESTED** [review]

**Review by alay10 — APPROVED** [review]
Looks good!

**Pull request merged**